src/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -7,6 +7,97 @@
 
 namespace System.Text.Json
 {
+    public partial class JsonArray : System.Text.Json.JsonNode, System.Collections.Generic.ICollection<System.Text.Json.JsonNode>, System.Collections.Generic.IEnumerable<System.Text.Json.JsonNode>, System.Collections.Generic.IList<System.Text.Json.JsonNode>, System.Collections.Generic.IReadOnlyCollection<System.Text.Json.JsonNode>, System.Collections.Generic.IReadOnlyList<System.Text.Json.JsonNode>, System.Collections.IEnumerable
+    {
+        public JsonArray() { }
+        public JsonArray(System.Collections.Generic.IEnumerable<bool> values) { }
```

Comment:
What about IEnumerable of decimal, DateTime{Offset}, Guid? It seems like we should support all the types that JsonDocument supports.

Maybe even IEnumerable of byte[] for base-64 encoded text.